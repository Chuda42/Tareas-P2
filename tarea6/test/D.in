asociarEnMap 5 15.0
desasociarEnMap 5
asociarEnMap 5 25.0
existeAsociacionEnMap 5
valorEnMap 5
estaLlenoMap
hacerVecinos 2 1 5.0
hacerVecinos 3 1 5.0
hacerVecinos 10 1 5.0
hacerVecinos 9 1 5.0
hacerVecinos 8 1 5.0
hayMParejas
sonVecinos 1 2
sonVecinos 6 1
distancia 1 2
vecinos 2
vecinos 9
vecinos 6
cantidadVertices
hacerVecinos 2 3 1.0
hacerVecinos 2 10 1.0
hacerVecinos 4 7 2.0
hacerVecinos 7 5 2.0
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
rangoCP
prioridad 3
actualizarEnCP 9 1.0
prioritario
Fin

