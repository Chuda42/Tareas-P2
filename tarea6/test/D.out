1>Establecida la asociación.
2>Eliminada la asociación.
3>Establecida la asociación.
4>Existe asociacion.
5>25.00
6>No lleno.
7>Nuevos vecinos: 2 1 5.00.
8>Nuevos vecinos: 3 1 5.00.
9>Nuevos vecinos: 10 1 5.00.
10>Nuevos vecinos: 9 1 5.00.
11>Nuevos vecinos: 8 1 5.00.
12>NO hay M parejas.
13>Son vecinos.
14>NO son vecinos.
15>5.00
16>1 
17>1 
18>
19>10.
20>Nuevos vecinos: 2 3 1.00.
21>Nuevos vecinos: 2 10 1.00.
22>Nuevos vecinos: 4 7 2.00.
23>Nuevos vecinos: 7 5 2.00.
24>Se insertó el elemento.
25>Se insertó el elemento.
26>Se insertó el elemento.
27>10.
28>3.00.
29>Se actualizó el valor.
30>9
31>Fin.
